client/src/App.js

```diff
@@ -10,7 +10,7 @@ import PostsListPage from './pages/PostsListPage';
 import PostFormPage from './pages/PostFormPage';
 import ShowPostPage from './pages/ShowPostPage';
 import AboutUsPage from './pages/AboutUsPage';
-
+import Login from './components/Login/Registration'
```

Comment:
Is there a benefit for having a subfolder for registration?

Reply:
The registration links the login and signup page, Login is the folder thats holds: Sign Up, Login , Registration